.github/actions/analyze-test-runs/action.yml

```diff
@@ -50,7 +51,7 @@ runs:
         # To cover cases where we use parameterized tests like
         # [WARNING]  io.camunda.zeebe.engine.state.BanInstanceTest.shouldBanInstance[PROCESS_MESSAGE_SUBSCRIPTION DELETE should ban instance true]
         # We grep the WARNING line and set the first argument to an empty string
-        flakyTests=$(grep "\[WARNING\] io.camunda" ${irOutputFile} | awk '{$1=""; print $0}')
+        flakyTests=$(grep -E "\[WARNING\] [a-z]+\." ${irOutputFile} | awk '{$1=""; print $0}')
```

Comment:
That is to match `io.atomix.raft.RaftSnapshotReplicationFailureHandlingTest.shouldNotRestartFromFirstChunkWhenInstallRequestTimesOut`

```sh
[WARNING] Flakes: 
[WARNING] io.atomix.raft.RaftSnapshotReplicationFailureHandlingTest.shouldNotRestartFromFirstChunkWhenInstallRequestTimesOut
[ERROR]   Run 1: RaftSnapshotReplicationFailureHandlingTest.shouldNotRestartFromFirstChunkWhenInstallRequestTimesOut:56 failing
[ERROR]   Run 2: RaftSnapshotReplicationFailureHandlingTest.shouldNotRestartFromFirstChunkWhenInstallRequestTimesOut:56 failing
[INFO]   Run 3: PASS
[INFO] 
```